test/conformance/kuttl/generate/clusterpolicy/cornercases/cpol-clone-sync-reinstall-policy/README.md

```diff
@@ -0,0 +1,13 @@
+## Description
+
+This is a corner case test to ensure a generate data rule can be triggered on the deletion of the trigger resource.
+
+## Expected Behavior
+
+1. when the trigger is created, the corresponding downstream target secret should be generated
+2. delete the policy, update the source, then re-install the policy with generateExisting=true, the change should be synced to the downstream target
+3. update the source again, the change should be synced to the downstream target
+
+## Reference Issue(s)
+
+https://github.com/kyverno/kyverno/issues/6398
```

Comment:
The referenced issue here doesn't appear to be correct. I think it's 7170, right?